www/include/options/accessLists/reloadACL/reloadACL.php

```diff
@@ -103,14 +119,20 @@
         $session_data[$cpt]["user_id"] = $r["user_id"];
         $session_data[$cpt]["user_alias"] = $rU["contact_name"];
         $session_data[$cpt]["admin"] = $rU["contact_admin"];
-        
-        $resCP = $pearDB->query("SELECT topology_name, topology_page, topology_url_opt FROM topology WHERE topology_page = '".$r["current_page"]."'");
```

Comment:
I think you can place this subquery directly in the first request to relieve the database (SELECT DISTINCT * FROM session)